pkg/openstack/openstack.go

```diff
@@ -129,7 +131,8 @@ type NetworkingOpts struct {
 
 // RouterOpts is used for Neutron routes
 type RouterOpts struct {
-	RouterID string `gcfg:"router-id"` // required
+	RouterID         string `gcfg:"router-id"` // required
```

Comment:
in my opinion this `required` should be removed because its not really required. Like mentioned in another PR - we have been using OCCM for 7 years without using this at all.